Automattic/wp-calypso
# Jetpack Onboarding: Update illustrations in JPO step 3

Resolves #23460.

This PR updates the two illustration in Step 3 of JPO so that the sizes match the other illustrations in the flow.

Before/After:

<img width="758" alt="screen shot 2018-03-20 at 3 09 34 pm" src="https://user-images.githubusercontent.com/204742/37683012-c414172c-2c50-11e8-9c93-a90a38c4f34d.png">


**Testing instructions:**
* Checkout this branch
* Pick a Jetpack site.
* Head to `/wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development` on the Jetpack site to start the onboarding flow.
* On Step 3, the site type step (homepage type), verify the illustrations look the same as the above "After" shot for Step 3.
